### ISSUE TITLE
Add hardhat task for `grantRole` to multiple address for the same role

### ISSUE THREAD

**Issue by 0x3bfc** [aurora-is-near/aurora-staking-contracts#93]
> LGTM with minor comments. For the CLAIM_ROLE and AIRDROP_ROLE, we will need several keys to have these roles (10 ?) so that the airdrop script can process requests in parallel. Should we use `_setRoleAdmin` to setup these keys separately or also grant these roles in the deployment ?

We can call `grantRole()` separately to add as many keys as we want. I will prepare that in a separate PR for the hardhat tasks.

_Originally posted by @0x3bfc in https://github.com/aurora-is-near/aurora-staking-contracts/issues/89#issuecomment-1113458498_
